api/envoy/config/cluster/aggregate/v2alpha/cluster.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package envoy.config.cluster.aggregate.v2alpha;
+
+option java_outer_classname = "ClusterProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.cluster.aggregate.v2alpha";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Aggregate cluster configuration]
+
+// Configuration for the aggregate cluster. See the :ref:`architecture overview
+// <arch_overview_aggregate_cluster>` for more information.
+message ClusterConfig {
+  // Load balancing clusters in aggregate cluster with explicit priority.
+  message LbCluster {
+    // Cluster name.
+    string name = 1;
+
+    // [#not-implemented-hide:] Cluster priority. If no priority is specified, the clusters will be
+    // prioritized by the ordering in :ref:`list
+    // <envoy_api_field_config.cluster.aggregate.v2alpha.ClusterConfig.clusters>`
+    uint32 priority = 2;
```

Comment:
You can't make a field a new field required without a sensible default, as this would break backwards compatibility. Can we just figure out some behavior in which the default here (which is zero) matches the existing scheme, i.e. this this what I mean about degenerate behavior?

Given, as discussed, it would be best to do explicit priorities in a follow PR, I think for this PR, you can basically just make it simple (but not very functional) by saying "every cluster has the same priority".

Reply:
The default behavior in this PR should be ordering the cluster by the position in the list implicitly. If this is the case,  I don't think we need `priority` any more. The aggregate cluster will always order the cluster in some way (either priority or the ordering in the list). Why not make the interface clear? 

After removing the priority, every cluster will only be prioritized by the position in the list and this also prevents clusters from having same priorities.